README.md

```diff
@@ -1,4 +1,5 @@
 # CppCoro - A coroutine library for C++
+#Coroutine Library for C++
```

Comment:
This seems to be repeating the first line above.
It's not clear what the purpose of this change is.
Can you elaborate?